docs/agents.md

```diff
@@ -108,7 +108,7 @@ You can also pass messages from previous runs to continue a conversation or prov
 
 #### Usage Limits
 
-PydanticAI offers a [`settings.UsageLimits`][pydantic_ai.settings.UsageLimits] structure to help you limit your
+PydanticAI offers a [`UsageLimits`][pydantic_ai.usage.UsageLimits] structure to help you limit your
```

Comment:
Can we cherry pick out the change that moves this to usage?

Reply:
I'm not sure it's worth it.